base/array.jl

```diff
@@ -343,7 +343,7 @@ fill(v, dims::Integer...) = fill!(Array{typeof(v)}(uninitialized, dims...), v)
     zeros([T=Float64,] dims...)
 
 Create an `Array`, with element type `T`, of all zeros with size specified by `dims`.
-See also [`ones`](@ref), [`similar`](@ref).
+See also [`fill`](@ref), [`ones`](@ref).
```

Comment:
Why ref `fill` and `ones` (rather than `fill` and `similar`)?

Reply:
The reference to similar here should have been removed as part of  #24656/#24863. I assume the reason we referenced it here was because previously we had both `zeros(A)` and `similar(A)` doing similar things for `A::AbstractArray`.